helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -431,6 +431,57 @@ clusterReceiver:
   # Once the receiver is stabilized, it'll be enabled by default in this helm chart
   eventsEnabled: false
 
+  # This flag enables Kubernetes objects collection using OpenTelemetry Kubernetes Object Receiver
+  # https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sobjectsreceiver
+  # This option requires `logsEnabled` to be set to `true` for either `splunkObservability` or `splunkPlatform`
+  # depending on where you want to send the events. Otherwise, this option will not have any effect.
+  # The receiver currently is in alpha state which means that events format might change over time.
+  # Once the receiver is stabilized, it'll be enabled by default in this helm chart
+  objectsEnabled: false
```

Comment:
Do we need this option? Can we use `k8sObjects` just instead given that it's empty by default?

Reply:
Ok, I deleted `objectsEnabled`